PowerShell/SFNuGetModule.psm1

```diff
@@ -25,22 +25,18 @@ function New-ServiceFabricNuGetPackage {
             Revision History:
                 10/09/2017 : Haishi Bai - Created.
     #>
+    [cmdletbinding()]
     param(
-        [string] $InputPath,
-        [string] $OutPath,
-        [switch] $Publish=$false
+        [parameter(mandatory=$true)][string] $InputPath,
+        [parameter(mandatory=$true)][string] $OutPath,
+        [parameter(mandatory=$false)][switch] $Publish=$false
     )
+
+    $ErrorActionPreference = "stop"
     
     #chek if InputPath exists
-    if (!$InputPath -Or !(Test-Path $InputPath)) {
-        Write-Host "Input path is null or not found."
-        Exit 1
-    }
-    
-    #check if OutPath parameter is null
-    if (!$OutPath) {
```

Comment:
The OutPath check is removed?

Reply:
Nope. The check if it is $null and then exiting is removed because now the header is doing this for you with "[parameter(mandatory=$true)][string] $OutPath"

The mandatory forces the caller to enter a value of no one is there and won't let the script run unless something is provided.

Reply:
i see!